requirements.txt

```diff
@@ -0,0 +1,10 @@
+notebook==5.7.10
+tornado==4.5.3
+rdflib==5.0.0
+graph-notebook==3.0.2
+gremlinpython==3.4.12
+requests==2.26.0
+backoff==1.11.1
+cchardet==2.1.7
+aiodns==3.0.0
+idna-ssl==1.1.0
```

Comment:
I'd split the dependencies in two blocks. The ones corresponding to `graph-notebook` and those corresponding to `neptune-python-utils`. I'd also add a comment before each block.

Reply:
Ok, It's a good improvement I will do it.